core/src/test/scala/stryker4s/Stryker4sTest.scala

```diff
@@ -48,5 +48,28 @@ class Stryker4sTest extends Stryker4sSuite {
                   Killed(Mutant(3, _, _, _), `expectedPath`)) =>
       }
     }
+
+    it("should log a warning when JVM max memory is too low") {
+      implicit val conf: Config = Config()
+      val testSourceCollector = new TestSourceCollector(Seq())
+      val testProcessRunner = TestProcessRunner()
+      val testMutantRunner = new ProcessMutantRunner(Command("foo", "test"), testProcessRunner)
+      val testReporter = new TestReporter
+
+      val sut = new TestStryker4s(
+        jvmEnoughMemory = false,
+        testSourceCollector,
+        new Mutator(new MutantFinder(new MutantMatcher), new StatementTransformer, new MatchBuilder),
+        testMutantRunner,
+        testReporter
+      )
+
+      sut.run()
+
+      "The JVM doesn't have a lot of memory assigned to it. " +
```

Comment:
We should also test if the JVM does have enough memory, that the warning is __not__ logged